.github/ISSUE_TEMPLATE/request-access.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Request access
+about: Request access to the IBM.com Expressive and IBM.com Library design kits
+title: Design kit access request
+labels: adopter support
+assignees: ljcarot, wonilsuhibm, oliviaflory
+
+---
+
+Thanks for your interest in the IBM.com Design kits!
+
+We are in a closed adoption phase at the moment, please answer the following questions to help us better understand your project and someone from the team will reach out to you soon.
```

Comment:
We are in a closed adoption phase at the moment.  Please answer the following questions to help us better understand your interest.  Someone from DDS will reach out to you soon regarding your request for access.

Reply:
Since we are not DDS anymore, I replaced DDS with Carbon for IBM.com

Reply:
This has been updated